tests/publiccloud/boottime.pm

```diff
@@ -297,6 +297,7 @@ sub measure_timings {
     }
 
     $ret->{analyze}->{ssh_access} = $instance->wait_for_ssh(timeout => 300);
+    assert_script_run(sprintf('ssh-keyscan %s >> ~/.ssh/known_hosts', $instance->public_ip));
```

Comment:
exactly ! :)

Reply:
I re-added it and don't wanna hear now any comments about duplication here *pointing at the last PR I made here* :wink:

Reply:
my bad , I was not realizing what price we were paying here :)